unittests/delay_tests.cpp

```diff
@@ -2339,8 +2302,9 @@ BOOST_AUTO_TEST_CASE( max_transaction_delay_create ) { try {
                         ("auth",  authority(chain.get_public_key(tester_account, "first"), 50*86400)) ), // 50 days delay
       chain::action_validate_exception,
       [&](const chain::transaction_exception& ex) {
-         string expected = "message validation exception (3040000)\nCannot set delay longer than max_transacton_delay, which is 3888000 seconds";
-         return expected == string(ex.to_string()).substr(0, expected.size());
+         string check_str = "Cannot set delay longer than max_transacton_delay";
+         BOOST_ASSERT(e.to_detail_string().find(check_str) != std::string::npos);
```

Comment:
BOOST_ASSERT is compiled out in release build. This doesn't even build in debug.